packages/@aws-cdk/cloud-assembly-schema/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@aws-cdk/cloud-assembly-schema",
+  "version": "0.0.0",
+  "description": "Cloud Assembly Schema",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "java": {
+        "package": "software.amazon.awscdk.cloudassembly.schema",
+        "maven": {
+          "groupId": "software.amazon.awscdk",
+          "artifactId": "cdk-cloud-assembly-schema"
+        }
+      },
+      "dotnet": {
+        "namespace": "Amazon.CDK.CloudAssembly.Schema",
+        "packageId": "Amazon.CDK.CloudAssembly.Schema",
+        "signAssembly": true,
+        "assemblyOriginatorKeyFile": "../../key.snk",
+        "iconUrl": "https://raw.githubusercontent.com/aws/aws-cdk/master/logo/default-256-dark.png"
+      },
+      "python": {
+        "distName": "aws-cdk.cloud-assembly-schema",
+        "module": "aws_cdk.cloud_assembly_schema"
+      }
```

Comment:
Find naming across quite inconsistent.

If we pick 'cloudassembly' as one word and 'schema' as another word. We should keep that consistent everywhere, no?

For instance, maven should be "cdk-cloudassembly-schema" and python - "aws-cdk.cloudassembly-schema"

Reply:
I tend to agree, and this was my initial though as well. Eventually I ended up doing the same thing as `cdk-assets-schema`. Going to keep this as is unless you feel strongly about it.

Reply:
To clarify, my suggestion is to change maven artifactId to `cdk-cloudassembly-schema` and python distname and module name to `aws-cdk.cloudassembly-schema`